src/services/helperService.ts

```
@@ -17,6 +19,21 @@ export const decodeBase64ToJson = (input: string) => {
 };
 
 export const isBase64 = (str: string) => {
-    const base64Pattern = /^([A-Za-z0-9+/=]|\r|\n)+$/;
-    return base64Pattern.test(str);
-  }
\ No newline at end of file
+  const base64Pattern = /^([A-Za-z0-9+/=]|\r|\n)+$/;
+  return base64Pattern.test(str);
+}
+
+export const isDateString = (value: any): boolean => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)